docs/content/ingestion/index.md

```diff
@@ -88,7 +88,7 @@ If `type` is not included, the parser defaults to `string`.
 
 | Field | Type | Description | Required |
 |-------|------|-------------|----------|
-| type | String | This should say `protoBuf`. | no |
+| type | String | This should say `protobuf`. | no |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
```

Comment:
also need to add the doc for "descriptor" file and the fact that it has to be in the classpath